src/pages/auth/sign-in.tsx

```diff
@@ -100,7 +100,8 @@ export default function SignIn(props: PageProps) {
 
         <div className="mt-14 sm:mx-auto sm:w-full sm:max-w-[480px]">
           <div className="divide-y bg-white p-6 py-6 shadow sm:rounded-lg sm:px-12">
-            {props.authProviders.credentials ? (
+            {env.NEXT_PUBLIC_INTERNAL_CREDENTIALS_DISABLED !== "true" && props.authProviders.credentials
```

Comment:
Remove the environment condition as condition has moved to props

Reply:
[716807b](https://github.com/langfuse/langfuse/pull/510/commits/716807b6c110dd1f9d55ab9148cfaeb6e70473be)